packages/shared/args.js

```
@@ -0,0 +1,30 @@
+function args(args) {
```

`args(["hello", -c"])` returns `{c: undefined}` I propose this condition should throw an Error.